ruby/example.rb

```diff
@@ -4,7 +4,7 @@
 manifest = {
   :wasm => [{:path => "../wasm/code.wasm"}]
 }
-plugin = Plugin.new(manifest)
+plugin = Extism::Plugin.new(manifest)
```

Comment:
Perhaps the example in the documentation site needs to be changed as well? or is this links to that automatically?